plot-api/src/main/kotlin/jetbrains/letsPlot/scale/DateTime.kt

```diff
@@ -12,7 +12,23 @@ import jetbrains.datalore.plot.config.Option.Scale.DATE_TIME
 import jetbrains.letsPlot.intern.Options
 import jetbrains.letsPlot.intern.Scale
 
-
+/**
+ * Continuous position scale (x).
+ *
+ * @param name string
+ *      The name of the scale - used as the axis label or the legend title. If None, the default, the name of the scale
+ *      is taken from the first mapping used for that aesthetic.
+ * @param breaks list of numeric
+ *     A numeric vector of positions (of ticks).
+ * @param labels list of strings
+ *      A vector of labels (on ticks).
+ * @param limits list of numerics
+ *      A numeric vector of length two providing limits of the scale.
+ * @param expand
```

Comment:
List size == 1 ==> only multiplicative expand (and additive expand by default)

Reply:
Added to the description